blogposts/2021/release-3.28.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Sourcegraph 3.28 release"
+publishDate: 2021-05-20T10:00-07:00
+description: "Sourcegraph 3.28 release"
+tags: [blog, release]
+slug: "release/3.28"
+published: true
+changelogItems:
+  # Added
+  - description: "Added `select:commit.diff.added` and `select:commit.diff.removed` for `type:diff` search queries. These selectors return commit diffs only if a pattern matches in `added` (respectively, `removed`) lines."
+    url: https://github.com/sourcegraph/sourcegraph/pull/20328
+    category: Search
+  - description: "Additional language autocompletions for the `lang:` filter in the search bar."
```

Comment:
@rvantonder - what languages and/or how many languages did we add? I'd like to mention them here.

Reply:
Added 

```
"Assembly",
"Bash",
"Erlang",
"Elixir",
"Dart",
"Haskell",
"Kotlin",
"Julia",
"Objective-C",
"OCaml",
"Scala",
"SQL",
"VBA",
"XML",
"Zig",
```

Full list https://sourcegraph.com/github.com/sourcegraph/sourcegraph@75656d82c51cd628d11ddde15e0dbccd2a093766/-/blob/client/shared/src/search/query/filters.ts#L143-178